src/seedu/addressbook/data/person/PostalCode.java

```diff
@@ -0,0 +1,18 @@
+package seedu.addressbook.data.person;
+
+public class PostalCode {
+    private int postalCode;
+
+    public PostalCode(int postalCode) {
+        this.postalCode = postalCode;
+    }
+
+    public int getPostalCode() {
+        return postalCode;
+    }
+
+    @Override
```

Comment:
do you also need to override the `toString()` and `equals()` methods here?
think why or why not?